app/views/payment-links/edit.njk

```diff
@@ -38,6 +38,11 @@
       {{ product.price | penceToPoundsWithCurrency }}
     {% else %}
       <span class="pay-text-grey">User can choose</span>
+      {% if product.amountHint %}
+        <span class="govuk-!-display-block pay-text-grey">
+          {{ product.amountHint | striptags(true) | escape | nl2br }}
```

Comment:
What does `escape | nl2br` do?

Reply:
This is so we display back the hint correctly as the user entered it.
`escape` will escape special characters so they are displayed correctly when the HTML is rendered
`nl2br` converts new line characters to `<br>` tags so that line breaks put in by the user are displayed back correctly.

This is copied from above, where we display back the reference hint.